core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -906,6 +906,9 @@
     build01_build01_reg_auth_value.txt:
       bw_item: build_farm
       attachment: build01_build01_reg_auth_value.txt
+    build01_build01_reg_auth_value.txt:
```

Comment:
This needs to be `build01_build02_reg_auth_value.txt` and why does it have the `build01` prefix?

Reply:
`build01` tells the sa we used for generate the docker auth entry.
It was my bad idea at that time (See my PR summary).
But we name it for api.ci and build01 and it would take time to rename them all.

Reply:
The key here is the key in the resulting secret. Whats on line 909 will overwrite whats on line 906, as its the same key

Reply:
Just got your point.
My bad.
Updated now.